.circleci/config.yml

```diff
@@ -36,7 +48,8 @@ workflows:
           name: tag_version
           context: CodacyAWS
           requires:
-            - build-and-publish
+            - build-and-publish-2_12
+            - build-and-publish-2_11
```

Comment:
Can't you just have the build-and-publish-2_12 since it depends on the 2_11?

Reply:
I just separated them to avoid issues with sonatype. Ideally they could be parallel and independent.

Reply:
I understand, but as it is now we don't need to wait for both in the CI, since 2_12 only starts running after 2_11 is finished, so in this step technically we can wait only for the 2_12

Reply:
ahhh I see what you mean. I can do that if you really want

Reply:
done